operations/boot_orchestration/BOS_Workflows.md

```diff
@@ -1,39 +1,151 @@
+
+    In this example of booting nodes, the first phase is `powering-on`.  If queried at this point, the nodes will have a status of `power-on-pending`.
+
+1. **Power-on operator**
+
+    The power-on operator will detect nodes with a `power-on-pending` status. The power-on operator first sets the desired boot artifacts in BSS.
+    If configuration is enabled for the node, the power-on operator will also call CFS to set the desired configuration and disable the node with CFS.
+    The node must be disabled within CFS so that CFS does not try to configure node until it has booted.
+    The power-on operator then calls CAPMC to power-on the node.
+    Lastly, the power-on operator will update the state of the node in BOS, including setting the last action. If queried at this point, the nodes will have a status of `power-on-called`.
+
+1. **CAPMC boots nodes**
+
+    CAPMC interfaces directly with the Redfish APIs and powers on the selected nodes.
+
+1. **BSS interacts with the nodes**
+
+    BSS generates iPXE boot scripts based on the image content and boot parameters that have been assigned to a node. Nodes download the iPXE boot script from BSS.
+
+1. **Nodes request boot artifacts from S3**
+
+    Nodes download the boot artifacts. The nodes boot using the boot artifacts pulled from S3.
+
+1. **Status operator (configuring)**
+
+    The status operator monitors a node's power state until HSM reports that the power state is on.  
+    When the power state for a node is on, the status operator will either set the phase to `configuring` if CFS configuration is required or it will clear the current phase if the node is in its final state.
+
+1. **CFS applies configuration**
+
+    If needed, CFS runs Ansible on the nodes and applies post-boot configuration \(also called node personalization\).
+
+1. **Status operator (complete)**
+
+    The status operator will continue monitoring the states for each node until CFS reports that configuration is complete.
+    The status operator will clear the current phase now that the node is in its final state. The status operator will also disable components at this point.
+
+1. **Session completion operator**
+
+    When all nodes belonging to a session have been disabled, the session is marked complete, and its final status is saved to the database.
+
+### V2 Reboot Nodes
+
+**Use Case:** Administrator reboots and configures select compute nodes.
+
+**BOS V2 Reboot Flow Diagram:** This labels on the diagram correspond to the workflow steps listed below.
+
+![Boot Nodes](../../img/operations/boot_orchestration/bos_v2_reboot.png)
 
 **Workflow Overview:** The following sequence of steps occur during this workflow.
 
-1.  **Administrator creates a configuration**
+1. **Administrator creates a configuration**
 
     Add a configuration to CFS.
 
+    (`ncn-mw#`)
+
     ```bash
     cray cfs configurations update sample-config --file configuration.json
     ```
```

Comment:
I can't leave feedback on the output a few lines down, but the cloneUurl line is split across two lines, which makes the formatting poopy (technically speaking). Please reunite those two lines that were sadly split in twain.